libs/features/mas/src/variants/pricing-widget.css.js

```diff
@@ -0,0 +1,36 @@
+export const CSS = `
+:root {
+  	--merch-card-pricing-widget-width: 132px;
+  	--merch-card-pricing-widget-height: 212px;
+}
+
+merch-card[variant="pricing-widget"] [slot="heading-xxxs"] {
+    letter-spacing: normal;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--spectrum-gray-800);
+}
+
+merch-card[variant="pricing-widget"] [slot="body-xxs"] {
```

Comment:
are we duplicating all the styles here?
if the only thing that changes is the color, we can redefine the color variables here.

Reply:
In this particular card slot, there is a requirement to trim the text and display ellipsis when the content length is bigger than the space available `54px`.